### PR TITLE
Fixing codegen scripts 😅

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

<del>For some reason it fails on the `:latest` images…</del>

`go install` install binaries either in `$GOBIN` or
`$GOPATH/bin`. This means, you can do a `go install` and *hard code*
`$GOPATH/bin` to find the installed binaries.

As, in any case, this folder is usually in the `$PATH`, we can just
rely on that.

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

Double check this list of stuff that's easy to miss:

- If you are adding [a new binary/image to the `cmd` dir](../cmd), please update
  [the release Task](../tekton/publish.yaml) to build and release this image.

## Reviewer Notes

If [API changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md) are included, [additive changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#additive-changes) must be approved by at least two [OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS) and [backwards incompatible changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-incompatible-changes) must be approved by [more than 50% of the OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS), and they must first be added [in a backwards compatible way](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-compatible-changes-first).

### CODE REVIEW

File: hack/update-codegen.sh [hunk 1]
```diff
@@ -18,6 +18,8 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+set -x
```
Comment (chmouel) [inline hack/update-codegen.sh:21]: is that intentional to be kept (it's going to be more verbose)

Comment (vdemeester) [reply hack/update-codegen.sh:21]: right, I should remove it, but I wanted to make sure it passes before :sweat_smile:

**Comment by chmouel** [comment]
/lgtm

Looking Good! 🤙🏽

**Pull request merged**